src/infrastructure/selectors/IdentitySelectorAdapter.js

```diff
@@ -1,12 +1,16 @@
 /* eslint-disable no-unused-vars */
-const DataSelectorPort = require('../domain/common/port/DataSelectorPort');
+const DataSelectorPort = require('../../domain/common/port/DataSelectorPort');
 
 class IdentitySelectorAdapter extends DataSelectorPort {
   canHandle(contentType) {
     return contentType === 'ipfs';
   }
 
   select(data, path) {
+    if (path) {
```

Comment:
We should care about input correctness. If not, this could end up with some strange behaviors if for any reason we would like to change sth in the future.